py4DSTEM/io/nonnative/empad.py

```diff
@@ -0,0 +1,91 @@
+# Reads an EMPAD file
+#
+# Created on Tue Jan 15 13:06:03 2019
+# @author: percius
+# Edited on 20190409 by bsavitzky and rdhall
+# Edited on 20210628 by sez
+
+import numpy as np
+from pathlib import Path
+from ..datastructure import DataCube
+from ...process.utils import bin2D, tqdmnd
+
+from pdb import set_trace
+
+
+def read_empad(filename, mem="RAM", binfactor=1, metadata=False, **kwargs):
+    """
+    Reads the EMPAD file at filename, returning a numpy array.
+
+    EMPAD files are shaped as 130x128 arrays, consisting of 128x128 arrays of data followed by
+    two rows of metadata.  For each frame, its position in the scan is embedded in the metadata.
+    By extracting the scan position of the first and last frames, the function determines the scan
+    size. Then, the full dataset is loaded.
+
+    Note that the output of this function is a datacube which includes the 2 rows of metadata.
+
+    Accepts:
+        filename    (str) path to the empad file
+
+    Returns:
+        data        (ndarray) the 4D datacube, including the metadata.
+                    raw data only can be accessed data[:,:,:128,:]
```

Comment:
It says here that the fn returns the datacube *with* the 2 metadata lines, but the code (lines 64,68,72) do *not* appear to be including the metadata lines in the returned datacube.  I think this is probably the right way to do this, but we should then change the `Note` and `Returns` descriptions in this docstring